app/models/concerns/user_access_key_overrides.rb

```diff
@@ -23,6 +23,13 @@ def password=(new_password)
     write_legacy_password_attributes(digest)
   end
 
+  def authenticatable_salt
+    return if encrypted_password_digest.blank?
```

Comment:
I don't think we want to return here. There are probably some users who don't yet have an `encrypted_password_digest`, right? If those users sign in with RC 59, which uses `password_salt` as the `authenticatable_salt`, and then they hit RC 60, the salt will be different and their session will be dropped.

Reply:
There are not users who don't have an encrypted password digest. Those have been backfilled.

Reply:
Rather, there should not be users who have an encrypted password but not an encrypted password digest.